js/webviews.js

```diff
@@ -87,6 +87,20 @@ bindWebviewIPC('goForward', function () {
   })
 })
 
+bindWebviewIPC('updatedSettings', function () {
+  settings.clearCache()
+
+  darkModeUpdate()
```

Comment:
I don't think this is necessary here - clearCache will call load(), which will dispatch a settingsUpdated event, which will cause darkModeUpdate to happen anyway.